hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -373,4 +407,14 @@ private static UpdateMode getUpdateMode(final DmfAttributeUpdate update) {
         return null;
     }
 
-}
+    private boolean isMultiAssignmentsEnabled() {
+        return getConfigValue(MULTI_ASSIGNMENTS_ENABLED, Boolean.class, Boolean.FALSE);
+    }
+
+    private <T extends Serializable> T getConfigValue(final String key, final Class<T> valueType,
+            final T defaultValue) {
+        return systemSecurityContext
+                .runAsSystem(() -> tenantConfigurationManagement.getConfigurationValue(key, valueType, defaultValue));
```

Comment:
double-check if runAsSystem is really needed...

Reply:
I don't think so tenant context is already within the entry point `onMessage(...)`